.github/workflows/deploy.yml

```
@@ -66,7 +66,7 @@ jobs:
           tlmgr install capt-of caption ctex enumitem everysel fancyhdr fandol \
                         float ms needspace oberdiek parskip pdftexcmds tabulary \
                         titlesec ulem upquote varwidth wrapfig xcolor xecjk \
-                        xindy zhnumber
+                        xindy zhnumber collection-fontsrecommended
```

由于现在在编译 HTML 文档的时候也需要 LaTeX 了，所以需要把第 60-69 行移动到 57 行之前。

---

Done

---

把两类包分开这样好知道每个需求需要哪些包
```suggestion
                        xindy zhnumber
          # install packages for GMT LaTeX integration
          tlmgr install collection-fontsrecommended
```